tests/migrations/0001_initial.py

```diff
@@ -64,4 +64,27 @@ class Migration(migrations.Migration):
             },
             bases=(models.Model,),
         ),
+        migrations.CreateModel(
```

Comment:
Oops, you're right. Makes sense for the tests. Thanks. I'll have a look
at the PR tomorrow. Thank you for your contribution!

On Mon, Jun 06, 2016 at 11:23:49AM -0700, blag wrote:
>> @@ -64,4 +64,27 @@ class Migration(migrations.Migration):
>>              },
>>              bases=(models.Model,),
>>          ),
>> +        migrations.CreateModel(
>
>Usually I would agree with you, and I would never have done that for the project's main migrations. But those are the migrations for the test fixtures - that database is setup *from scratch* for every test and completely torn down afterward. Because of that, nobody should have any of those objects in their persistent database, so modifying that migration won't/shouldn't impact any users. Having multiple migrations for the test database is, as far as I know, pointless.
>
>Let me know if you still want me to create a new migration instead - I'm more interested in getting this merged.
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/MarkusH/django-osm-field/pull/9/files/4821f8b19435808048b1735568f11f713a1f1bbd#r65941875